test/sass.scss

```diff
@@ -35,9 +35,6 @@
 .d2l-test-heading-4 {
 	@include typography.d2l-heading-4();
 }
-.d2l-test-label-text {
-	@include typography.d2l-label-text();
```

Comment:
This Sass mixin is only ever used by buttons (and I'm going to remove that usage) and we're not actually testing this CSS class anyway.